src/aiosmtplib/connection.py

```diff
@@ -261,9 +261,9 @@ def source_address(self) -> str:
             response = await asyncio.wait_for(
                 waiter, timeout=self.timeout, loop=self.loop
             )
-        except asyncio.TimeoutError as exc:
+        except asyncio.TimeoutError:
             self.close()
-            raise SMTPTimeoutError(str(exc))
+            raise SMTPTimeoutError("Timeout on wait server readiness")
```

Comment:
I'm torn here if it makes sense to create a new class for `SMTPReadTimeoutError` but I think for now this is good. Maybe can you change the wording to:
"Timed out waiting for server ready message"

Reply:
I create new class. It seems to me that it will be more clearly